src/css.js

```diff
@@ -16,7 +16,7 @@ function css(val) {
                 ? // Tagged templates
                   compile(_val, [].slice.call(arguments, 1), ctx.p)
                 : // Regular arrays
-                  _val.reduce((o, i) => (i ? Object.assign(o, i.call ? i(ctx.p) : i) : o), {})
+                  _val.reduce((o, i) => (i ? { ...o, ...(i.call ? i(ctx.p) : i) } : o), {})
```

Comment:
It seems like as a compromise I would revert this line back, as this would most likely cause a perf drop, what do you think @chrstntdd ?